server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider using a descriptive variable name instead of a generic name like 'HOSTNAME'. This can help improve code readability and maintainability.